crates/bevy_ui/src/layout/convert.rs

```diff
@@ -133,6 +134,7 @@ pub fn from_style(context: &LayoutContext, style: &Style) -> taffy::style::Style
             .collect::<Vec<_>>(),
         grid_row: style.grid_row.into(),
         grid_column: style.grid_column.into(),
+        ..default()
```

Comment:
There are some new `overflow` and `scrollbar_width` properties. Should they be replicated on `bevy_ui::Style` and if so, should that be done in this PR?

Reply:
Yes please to both.